js/views/modals/orderDetail/OrderDetail.js

```diff
@@ -529,17 +529,21 @@ export default class extends BaseModal {
     let showDisputeOrderButton = false;
 
     if (this.buyerId === app.profile.id) {
+      console.log('moo');
```

Comment:
Just a reminder to take the console.logs out.

Reply:
Are you saying a stray 'moo' isn't useful?

Anyhow, for this one actually, I'd like to leave it in because this code is changed in the escrow timeout branch and I'm concerned the merge might be weird if I tweak it. I'll make sure the consoles are removed in the escrow branch.

Now if we end up releasing without the escrow branch, then I suppose I could remove the consoles and just deal with the likely merge conflicts.